server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -86,9 +88,19 @@ public long getTimeInMillis() {
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
-            out.writeVLong(current);
+            if (current < 0) {
+                logger.warn("'current' is negative [{}]", current);
+                out.writeVLong(0);
+            } else {
+                out.writeVLong(current);
+            }
             out.writeVLong(total);
-            out.writeVLong(timeInMillis);
+            if (timeInMillis < 0) {
+                logger.warn("'timeInMillis' is negative: [{}]", timeInMillis);
+                out.writeVLong(0);
+            } else {
+                out.writeVLong(timeInMillis);
+            }
```

Comment:
did we also see `timeInMillis` becomes negative? If not, why this change is required